packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -83,6 +83,10 @@ export default declare((api, options) => {
         }
       }
 
+      if (replacementPath.isOptionalCallExpression()) {
+        replacementPath.node.type = "CallExpression";
```

Comment:
Looks good, the type should be changed in the transform for sure? This logic is similar to what was done in https://github.com/babel/babel/pull/8210/files#diff-accf2589ebb96f2b32131e0a1f8a08eaR24

```
if (objectPath.isOptionalMemberExpression()) {
  objectPath.node.type = "MemberExpression";
}
```